.php_cs.dist

```diff
@@ -26,7 +26,7 @@ return PhpCsFixer\Config::create()
         ],
         'function_to_constant' => false,
         'no_alias_functions' => false,
-        'non_printable_character' => false,
+        'non_printable_character' => true,
```

Comment:
Maybe even better changing to:
```
'non_printable_character' => [
            'use_escape_sequences_in_strings' => true,
        ],
```
same as `true`, but if a non-printable character will be find by CS fixer, it will not remove it, but instead of it will escape it with Unicode escape like `\u{HEX}`.

Reply:
That's nice! Even better!! I just added it Thanks